timesketch/ui/views/home.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module implements HTTP request handler."""
+
+from flask import Blueprint
+from flask import render_template
+from flask import redirect
+from flask import request
+from flask import url_for
+from flask_login import current_user
+from flask_login import login_required
+from sqlalchemy import not_
+
+from timesketch.models.sketch import Sketch
+from timesketch.lib.forms import HiddenNameDescriptionForm
+from timesketch.models import db_session
+
+
+# Register flask blueprint
+home_views = Blueprint('home_views', __name__)
+
+
+@home_views.route('/', methods=['GET', 'POST'])
+@home_views.route('/sketch/', methods=['GET', 'POST'])
+@login_required
+def home():
+    """Handler for the home page view.
```

Comment:
Since this is a function I opt to make the docstring more like:
Generates the home page view template.